.styleci.yml

```diff
@@ -1,6 +1,8 @@
 php:
   preset: laravel
   version: 8.2
+  enabled:
```

Comment:
I could update the Laravel preset to include this, instead, if you'd prefer.

Reply:
That's would be great